### ISSUE TITLE
Possibility to export data in google authenticator compatible QR-code

### ISSUE THREAD

**Issue by vaminakov** [beemdevelopment/Aegis#551]
Hello! Now aegis support input google authenticator qr-codes, but doesnt support export of it.
It will be very usable to add this possibility. By the way, it will be cool to user can choose number of parts of QR-code (which doesnt support google authenticator).

**Comment by alexbakker** [comment]
Aegis already supports displaying QR codes for entries. Try long pressing on one and tapping the QR code button at the top right.

**Comment by vvl-rulez** [comment]
> Aegis already supports displaying QR codes for entries

One at a time, not all at a time and one QR code

> Could you rephrase that? I'm not sure what you mean.

This is a continuation of the phrase above. When you export data using qr code from google authenticator (for example I have 20 accounts) it makes 3 qr codes version 10, although in theory this data can fit into one qr code version 40. It would be cool if aegis could support the export of all data into one qr-code compatible with google authenticator, and you could choose a qr-code standard (for example, if you have many accounts, then one large qr-code version 40, or 3-4 small qr-codes version 10 etc)

**Comment by alexbakker** [comment]
We don't want to show QR codes that could be too large to scan. Implementing this the same way as Google Authenticator, with the same amount of entries in a QR code, sounds fine to us though. Is that something you'd like to work on?

**Comment by vvl-rulez** [comment]
> We don't want to show QR codes that could be too large to scan.

Maybe user can make a choice which size of QR-code he'll make?

> Is that something you'd like to work on?

Sure, I can do my best. But I doesnt code on Java, but I know QR standart and can test all stages of work.

**Comment by alexbakker** [comment]
> Maybe user can make a choice which size of QR-code he'll make?

I don't think exposing that level of customization desirable. Let's focus on implementing this so that it works for all users by default.

**Comment by MoralCode** [comment]
just dropping this tool here as its fairly similar to this idea but in the form of a script to decrypt backup files from another OTP app: https://github.com/CooperRS/decrypt-otpauth-files

**Comment by stavpup** [comment]
This feature is very much needed. Also there should be a tool tip that if you long press, you can access the QR code. I straggled to find this capability. Any progress ?

**Comment by alexbakker** [comment]
My bad, I misread your comment. Either way, this is kind of off topic for this issue. If you have a concrete proposal for what this tooltip should say and when/where it should be shown, feel free to open a separate issue and we'll discuss it there!

**Comment by alexbakker** [comment]
Implemented by #964.
